### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  com.microsoft.azure.toolkit.lib.auth.exception.AzureToolkitAuthenticationException: you are not signed-in.

### ISSUE THREAD

**Issue by f97one** [microsoft/azure-tools-for-java#5839]
IntelliJ build version: 2021.2.2 IU-212.5284.40
OS: Windows 10
JDK: JetBrains s.r.o. 11.0.12
Plugin version: 3.56.0-2021.2
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,8,1921x1040,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=com.microsoft.azure.toolkit.lib.auth.exception.AzureToolkitAuthenticationException: you are not signed-in.
	at com.microsoft.azure.toolkit.lib.auth.AzureAccount.lambda$account$0(AzureAccount.java:67)
	at com.microsoft.azure.toolkit.lib.auth.AzureAccount.account(AzureAccount.java:67)
	at com.microsoft.azure.toolkit.intellij.common.component.SubscriptionComboBox.<init>(SubscriptionComboBox.java:23)
	at com.microsoft.azure.toolkit.intellij.connector.database.DatabaseResourcePanel.$$$setupUI$$$(DatabaseResourcePanel.java)
	at com.microsoft.azure.toolkit.intellij.connector.database.DatabaseResourcePanel.<init>(DatabaseResourcePanel.java:67)
	at com.microsoft.azure.toolkit.intellij.connector.database.MySQLDatabaseResourcePanel.<init>(MySQLDatabaseResourcePanel.java:27)
	at com.microsoft.azure.toolkit.intellij.connector.mysql.MySQLD

**Comment by andxu** [comment]
It is fixed in 3.57.0, can you verify it now?

**Comment by wangmingliang-ms** [comment]
please update to latest version
